ui/apps/everest/src/components/time-selection/time-selection.constants.ts

```diff
@@ -12,6 +12,23 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+import {
```

Comment:
as we don't need anymore data for time selection in DbWizard all timeselection things were moved to timeselection component